index.js

```diff
@@ -38,6 +38,10 @@ function create (delay) {
     tape.skip(description, testFn)
   }
 
+  queueTest.slow = function (description, testFn) {
+    return create(500)(description, testFn)
```

Comment:
I think it's just:

```suggestion
    return create(500)
```

See `queueTest.delay`

Reply:
I think the problem that I mentioned in the issue was introduced by my self. Test pass now.

If return `create(500)` instead of `create(500)(description, testFn)` we should use the helpers as `vhs.slow()('describe the test', ...)` instead of `vhs.slow('describe the test', ...)`